mappings/net/minecraft/world/level/LevelAccessor.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5423 net/minecraft/world/level/LevelAccessor
```

Comment:
Yeah, this is just wrong, and sadly rasies my suspicions as to where you got this name from. For the sake of the yarn project I dont think I feel comfortable accepting this PR.

Thanks for understanding.